test/test_write.ml

```diff
@@ -50,8 +53,8 @@ let seq_to_file () =
        but we don't have a 'testable' instance for JSON values. *)
     Alcotest.fail "seq_{to,from}_file roundtrip failure"
 
-let single_json = [
+let single_json =
+  to_file_tests @ [
   "to_string", `Quick, to_string;
-  "to_file", `Quick, to_file;
   "seq_to_file", `Quick, seq_to_file;
```

Comment:
Maybe we can add small tests for `to_string` and `seq_to_file` wrt `suf` as well?

Reply:
I made all of them have tests for `?suf` `None`, `Some "\n"` and `Some ""`.

Also cleaned up the roundtrip test, since we have a `Yojson.Safe.t Testable.t` now I avoided the polymorphic comparison.